modules/nf-core/stardist/main.nf

```diff
@@ -0,0 +1,49 @@
+process STARDIST {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "ghcr.io/schapirolabor/stardist:0.9.1"
+
+    input:
+    tuple val(meta), path(image)
+
+    output:
+    tuple val(meta), path("*.stardist.tif"), emit: mask
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args    = task.ext.args   ?: ''
+    def prefix  = task.ext.prefix ?: "${meta.id}"
+    def VERSION = "0.9.1"
+
+
+
+    """
+    stardist-predict2d \\
+        -i $image \\
+        -o . \\
+        $args
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        stardist: $VERSION
```

Comment:
please also print the versions of python, tensorflow and tifffile

Reply:
Added

Reply:
nope not like this, but like this: https://nf-co.re/docs/guidelines/components/modules#emission-of-versions

Reply:
Alright, I fix the emission of versions 😄